src/python/grpcio/grpc/__init__.py

```diff
@@ -833,7 +833,11 @@ class HandlerCallDetails(six.with_metaclass(abc.ABCMeta)):
 
 
 class GenericRpcHandler(six.with_metaclass(abc.ABCMeta)):
```

Comment:
I think we _need_ to make ServiceRpcHandler a subinterface of GenericRpcHandler.  All the places where this would be used expect a GenericRpcHandler type.